### PR TITLE
metal: Remove all e2e-metal jobs

### PR DESCRIPTION
**Pull request by rna-afk**
Currently removing all jobs that refer to the cluster metal e2e
yaml file that has the template configuration to create clusters
in the baremetal environment.

### CODE REVIEW

**Comment by patrickdillon** [comment]
release-controller-config is failing with:
```
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.10.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.10
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.11.json:metal:release-openshift-ocp-installer-e2e-metal-4.11
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.11.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.11
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.12.json:metal:release-openshift-ocp-installer-e2e-metal-4.12
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.12.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.12
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.2.json:metal:release-openshift-ocp-installer-e2e-metal-4.2
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.2.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.2
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.3.json:metal:release-openshift-ocp-installer-e2e-metal-4.3
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.3.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.3
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.4.json:metal:release-openshift-ocp-installer-e2e-metal-4.4
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.4.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.4
[2022-06-30 20:35:00,979] Unable to locate job definition for: release-ocp-4.5.json:metal:release-openshift-ocp-installer-e2e-metal-4.5
[2022-06-30 20:35:00,980] Unable to locate job definition for: release-ocp-4.5.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.5
[2022-06-30 20:35:00,980] Unable to locate job definition for: release-ocp-4.6.json:metal:release-openshift-ocp-installer-e2e-metal-4.6
[2022-06-30 20:35:00,980] Unable to locate job definition for: release-ocp-4.6.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.6
[2022-06-30 20:35:00,980] Unable to locate job definition for: release-ocp-4.7.json:metal:release-openshift-ocp-installer-e2e-metal-4.7
[2022-06-30 20:35:00,980] Unable to locate job definition for: release-ocp-4.7.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.7
[2022-06-30 20:35:00,980] Unable to locate job definition for: release-ocp-4.8.json:metal:release-openshift-ocp-installer-e2e-metal-4.8
[2022-06-30 20:35:00,980] Unable to locate job definition for: release-ocp-4.8.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.8
[2022-06-30 20:35:00,980] Unable to locate job definition for: release-ocp-4.9.json:metal:release-openshift-ocp-installer-e2e-metal-4.9
[2022-06-30 20:35:00,980] Unable to locate job definition for: release-ocp-4.9.json:metal-serial:release-openshift-ocp-installer-e2e-metal-serial-4.9
```

It's unclear to me why these job definitions are expected. I could not find any dangling references, but I'm probably looking in the wrong place.

**Comment by rna-afk** [comment]
good to go 

/cc @patrickdillon

**Comment by patrickdillon** [comment]
/approve

reaching out to test platform team for further approval

File: ci-operator/jobs/openshift/installer/openshift-installer-release-4.6-presubmits.yaml [hunk 1]
```diff
@@ -1627,6 +1627,7 @@ presubmits:
           secretName: registry-pull-credentials
     trigger: (?m)^/test( | .* )e2e-libvirt,?($|\s.*)
   - agent: kubernetes
+<<<<<<< HEAD
```
Comment (patrickdillon) [inline ci-operator/jobs/openshift/installer/openshift-installer-release-4.6-presubmits.yaml:1630]: rebasing badness snuck in here

**Pull request merged**